src/app/components/client/stories/DataBrokerProfiles.stories.ts

```diff
@@ -0,0 +1,455 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { DataBrokerProfiles } from "../DataBrokerProfiles";
+import { ScanResult } from "../../../functions/server/onerep";
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
+const meta: Meta<typeof DataBrokerProfiles> = {
+  title: "DataBrokerProfiles",
+  component: DataBrokerProfiles,
+};
+export default meta;
+type Story = StoryObj<typeof DataBrokerProfiles>;
+
+const MockScanResults = [
```

Comment:
Instead of hardcoding 10(?) results here, do we want to use mocks from https://github.com/mozilla/blurts-server/blob/af9ee12e94844cd83408b38cd2c192805f44d746/src/apiMocks/mockData.ts#L29-L54

We even have this utility fn that returns an array with `3` scan results, but should be easy enough to configure to return a optional variable number https://github.com/mozilla/blurts-server/blob/af9ee12e94844cd83408b38cd2c192805f44d746/src/apiMocks/mockData.ts#L17-L22

Using the mock factory would reduce this file by like 431 lines of code.

Reply:
Love this.